package.json

```
@@ -1,5 +1,5 @@
 {
-  "version": "9.1.7",
+  "version": "9.1.8",
```

```suggestion
  "version": "9.2.0",
```

Can be marked as a minor update